### ISSUE TITLE
gorename: couldn't load packages due to errors

### ISSUE THREAD

**Issue by Gys** [redefiance/atom-go-rename#5]
Selected a symbol and trying to change it using Alt-r.
After about 2 seconds I get a red popup in top right corner:
gorename: couldn't load packages due to errors: bitbucket.org/xxx/yyy
Where bitbucket.org/xxx/yyy is a reference to the project folder in which I am currently working on main.go

Atom 1.02
Installed atom-go-rename
Installed gorename

**Comment by zkirill** [comment]
Run `go get golang.org/x/tools/cmd/gorename`.

**Comment by dcbishop** [comment]
I'm getting this error too.

gorename is installed.

**Comment by jwatte** [comment]
Yeah, this is annoying -- something is wrong, but without a more detailed log message, it's impossible to know exactly what is failing and needs fixing.
